<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>122</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>363</width>
    <height>122</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,6">
   <item row="0" column="1">
    <widget class="QWidget" name="w_loginSetting" native="true"/>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="cb_loginType">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAfterCurrent</enum>
       </property>
       <item>
        <property name="text">
         <string>单线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>离轴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>扫地机</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
